utils/uninstall.sh

```diff
@@ -248,9 +248,8 @@ main() {
     _shell_="${SHELL#${SHELL%/*}/}"
     _shell_rc=".""$_shell_""rc"
 
-    line_num="$(grep -n ". \"${IPATH}/env\"" "${__HOME__}/${_shell_rc}" | cut -d : -f 1)"
-
-    [ "$line_num" != "" ] && sed -i.wasmedge_backup -e "${line_num}"'d' "${__HOME__}/${_shell_rc}" ||
+    [[ -f "${__HOME__}/${_shell_rc}" ]] && line_num="$(grep -n ". \"${IPATH}/env\"" "${__HOME__}/${_shell_rc}" | cut -d : -f 1)" &&
+        [ "$line_num" != "" ] && sed -i.wasmedge_backup -e "${line_num}"'d' "${__HOME__}/${_shell_rc}" ||
```

Comment:
It still shows on MacOS:

```bash
Sourcing not found in /Users/q82419/.bashrc
```

But the `.bash_profile` is restored correctly.

Reply:
Sorry, removed the unnecessary warning.

Reply:
Thanks :+1: